File: lib/models/plan.dart
class PlanModel {
  int id, userId;
  String date;
  String name;
  dynamic set, reps, weight;
  bool isConducted;

  PlanModel.fromDB(Map<String, dynamic> db)
      : id = db['id'],
        userId = db['userId'],
        date = db['createdAt'].toString(),
        name = db['name'],
        set = db['set'],
        reps = db['reps'],
        weight = db['weight'],
        isConducted = db['isConducted'];
}

File: models/plan.js
const Sequelize = require("sequelize");

const sequelize = require("../util/database");

const Plan = sequelize.define("plan", {
  id: {
    type: Sequelize.INTEGER,
    autoIncrement: true,
    allowNull: false,
    primaryKey: true,
  },
  userId: { type: Sequelize.INTEGER, allowNull: false },
  name: { type: Sequelize.STRING },
  set: { type: Sequelize.INTEGER },
  reps: { type: Sequelize.INTEGER },
  weight: { type: Sequelize.DOUBLE },
  isConducted: { type: Sequelize.BOOLEAN },
});
module.exports = Plan;
